extra/mariabackup/xtrabackup.cc

```diff
@@ -271,6 +271,9 @@ static char *xtrabackup_debug_sync = NULL;
 
 my_bool xtrabackup_incremental_force_scan = FALSE;
 
+/* Ignore currupt pages (disabled by default). */
+ulong xtrabackup_innodb_force_recovery = 0;
```

Comment:
I think that we can just use the existing global variable `srv_force_recovery`.

Reply:
Yes, I initially thought that it would work, but when testing locally, it doesn't seem to work if the configuration is set in `backup-my.cnf`. I think the command line argument overrides the value set in the configuration file (only in `--innobackupex` mode), and it probably happens here:
https://github.com/MariaDB/server/blob/c0ca3c4ffa45042d608e55326036443e8c4f600d/extra/mariabackup/xtrabackup.cc#L6532-L6538

Also, added a comment about that [here](https://github.com/MariaDB/server/pull/1772/files#diff-d2ff45ced531afaa25c4b8fd289f51d0de59a96907894dc5b13807cbbba31d74R2272-R2277).

Reply:
@clickyotomy Thank you for clarifying this. I was not aware of this. @vlad-lesin @vaintroub do you have a comment? You have worked more on this code base than me.

However, I wonder if we could do with just one additional global variable that would be shared between `xtrabackup.cc` and `innobackupex.cc`.

Reply:
@dr-m,
> However, I wonder if we could do with just one additional global variable that would be shared between `xtrabackup.cc` and `innobackupex.cc`.

Yes, that will work!